bin/compile

```diff
@@ -145,7 +150,6 @@ if [ ! -z "$env_dir" ]
 then
     mkdir -p "$env_dir"
     env_dir=$(cd "$env_dir/" && pwd)
-    ln -sfn $build /app/code
```

Comment:
If we drop this, we should make sure that https://github.com/heroku/heroku-buildpack-go#using-with-cgo is still accurate or adjust it.